forc-plugins/forc-client/src/util/gas.rs

```diff
@@ -55,56 +51,3 @@ pub(crate) async fn get_script_gas_used(mut tx: Script, provider: &Provider) ->
         .await?;
     Ok(estimated_tx_cost.gas_used)
 }
-
-/// Returns an estimation for the max fee of `Create` transactions.
-/// Accepts a `tolerance` which is used to add some safety margin to the estimation.
-/// Resulting estimation is calculated as `(dry_run_estimation * tolerance)/100 + dry_run_estimation)`.
-pub(crate) async fn get_estimated_max_fee(
```

Comment:
Is this done automatically by the SDK now?

Reply:
Yes